MAKEWINDOWS.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for getting `make` to work on Windows
+
+The GNU command `make` does not work on Windows out of the box. Here are the steps I took to get it to work on my Windows 10 machine:
```

Comment:
Please don't use "I" and instead write something generic like: "Here are some possible steps that can make it to work on Windows 10"

Reply:
Well, the point was that this worked on my machine, but I can't guarantee it's gonna work on anyone else's. But okay, I will rewrite